evalml/pipelines/utils.py

```diff
@@ -84,8 +84,10 @@ def _get_preprocessing_components(
     if len(text_columns) > 0:
         pp_components.append(TextFeaturizer)
 
-    index_columns = list(X.ww.select("index", return_schema=True).columns)
-    if len(index_columns) > 0:
+    index_and_unknown_columns = list(
+        X.ww.select(["index", "Unknown"], return_schema=True).columns
```

Comment:
super-nit: consistency in the capitalization of 'index' and 'unknown'...all one way or another, if you please.